### PR TITLE
Cli user docs

### PR DESCRIPTION
**Pull request by will-moore**
Update install page and fix ```bin/omero``` -> ```omero``` in cli docs.

### CODE REVIEW

File: omero/users/cli/installation.rst [hunk 1]
```diff
@@ -1,43 +1,19 @@
 Installation
 ------------
 
-.. note:: The end of Windows support for OMERO.server means that the CLI is
-    unsupported on this platform too.
+.. note:: The the CLI is currently untested on Windows
+    but may be supported in future.
```
Comment (joshmoore) [inline omero/users/cli/installation.rst]:
```suggestion
    but may be supported in the future.
```

**Review by joshmoore — COMMENTED** [review]
One minor typo that needs correcting, but then I think getting this in makes good sense.

**Comment by will-moore** [comment]
Travis is failing on finding redirects:
```
     [exec] Adding redirects:
     [exec] 
     [exec] Exception occurred:
     [exec]   File "/home/travis/virtualenv/python3.6.7/lib/python3.6/shutil.py", line 121, in copyfile
     [exec]     with open(dst, 'wb') as fdst:
     [exec] FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/build/ome/ome-documentation/omero/_build/html/sysadmins/server-overview.html'
```
This was changed recently, https://github.com/ome/ome-documentation/pull/2042/commits/2e2202c2d7867fafb8592c191c9c8b086f55de31 cc @joshmoore

**Comment by joshmoore** [comment]
Redirect is fixed with my last PR. New failure is:

```
     [exec] Warning, treated as error:
     [exec] /home/travis/build/ome/ome-documentation/omero/users/clients-overview.rst:2:Duplicate explicit target name: "omero.figure".
```

**Comment by joshmoore** [comment]
Nice. Thanks!

**Comment by will-moore** [comment]
I have no idea what ```Duplicate explicit target name: "omero.figure"``` failure meant, but it drew my attention to the excess paragraph.

**Comment by joshmoore** [comment]
>  `Duplicate explicit target name: "omero.figure"`

Pretty sure it just means you linked to the same thing twice.

**Pull request merged**